src/mock-doc/element.ts

```diff
@@ -297,6 +314,25 @@ export class MockSVGElement extends MockElement {
   getTotalLength(): number {
     return 0;
   }
+  getComputedTextLength(): number {
+    return 0;
+  }
+  createSVGPoint() {
+    return {
+      matrixTransform: defaultSVGMatrix,
+      x: 0,
+      y: 0,
+    };
+  }
+  getBBox() {
+    return { x: 0, y: 0, width: 10, height: 10 };
+  }
+  getCTM() {
+    return defaultSVGMatrix;
+  }
+  getScreenCTM() {
+    return defaultSVGMatrix;
+  }
```

Comment:
Can we add return types to the function signatures added here please, since they don't return `undefined`/`void`?

Reply:
Sure but can I use named types like `DOMMatrix`, or `DOMRect`? or should the return types be anonymous? I'm not sure what's available while running in the testing context.

Reply:
I think you can, there's prior art here in this file for using types like `DOMPoint` and `SVGSVGElement`